docs/faq.md

```diff
@@ -35,6 +35,13 @@ This video which explains the basics of micro frontend architecture and how it c
 
 No, Luigi itself does not deliver any micro frontends. It is a framework that helps you develop micro frontends and connect them to web applications.
 
+### I don't want to use the default Fiori Fundamentals style. How can I style Luigi differently? How can I change the look of Luigi's UI?
+
+There are a few options to do that at the moment:
+- Use the Fundamental Styles theming capabilities which already allow you to achieve a lot by customizing the CSS variables. Find more info [here](https://github.com/SAP/theming-base-content).
+- Manually overwrite the styles where needed. The documentation page you are on right now can be used as an example, as it was developed with Luigi!
```

Comment:
I'd suggest to remove "!" at the end of the statement since you never know was it an excitement sign or a sort of screaming. I am never sure about the perception of people reading it behind their screens.